src/Support/DataClass.php

```diff
@@ -43,9 +44,9 @@ public function __construct(
 
     public static function create(ReflectionClass $class): self
     {
-        $attributes = collect($class->getAttributes())->map(
-            fn (ReflectionAttribute $reflectionAttribute) => $reflectionAttribute->newInstance()
-        );
+        $attributes = collect($class->getAttributes())
+            ->reject(fn (ReflectionAttribute $reflectionAttribute) => Str::startsWith($reflectionAttribute->getName(), 'JetBrains\\PhpStorm\\'))
```

Comment:
Can't we just do a class_exists($reflectionAttribute->getName()) here? This would rule out all non existing attributes

Reply:
fixed